modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -179,7 +179,11 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 						<liferay-ui:error exception="<%= NoSuchFormInstanceException.class %>" message="the-selected-form-no-longer-exists" />
 						<liferay-ui:error exception="<%= NoSuchStructureException.class %>" message="unable-to-retrieve-the-definition-of-the-selected-form" />
 						<liferay-ui:error exception="<%= NoSuchStructureLayoutException.class %>" message="unable-to-retrieve-the-layout-of-the-selected-form" />
-						<liferay-ui:error exception="<%= ObjectEntryValuesException.class %>" message="the-maximum-length-is-280-characters-for-text-fields" />
+						<liferay-ui:error exception="<%= ObjectEntryValuesException.IntegerSizeExceeded.class %>" message="object-entry-value-exceeds-integer-field-allowed-size" />
+						<liferay-ui:error exception="<%= ObjectEntryValuesException.LongMaxSizeExceeded.class %>" message="object-entry-value-exceeds-maximum-long-field-allowed-size" />
+						<liferay-ui:error exception="<%= ObjectEntryValuesException.LongMinSizeExceeded.class %>" message="object-entry-value-exceeds-minimum-long-field-allowed-size" />
+						<liferay-ui:error exception="<%= ObjectEntryValuesException.LongSizeExceeded.class %>" message="object-entry-value-exceeds-integer-field-allowed-size" />
```

Comment:
Can you change the message to match the exception, please?

Reply:
Thanks for noticing it!! Fixing right now!